### PR TITLE
DOC Use 'Axes' instead of 'axes' in axes._base.py

### PR DESCRIPTION
**Pull request by timhoffm**
## PR Summary

Towards #18726.

### CODE REVIEW

File: lib/matplotlib/axes/_base.py [hunk 1]
```diff
@@ -563,26 +563,26 @@ def __init__(self, fig, rect,
...
                  ):
         """
-        Build an axes in a figure.
+        Build an Axes in a figure.
 
         Parameters
         ----------
         fig : `~matplotlib.figure.Figure`
-            The axes is build in the `.Figure` *fig*.
+            The Axes is build in the `.Figure` *fig*.
```
Comment (story645) [inline lib/matplotlib/axes/_base.py:571]:
```suggestion
            The Axes is built in the `.Figure` *fig*.
```

File: lib/matplotlib/axes/_base.py [hunk 2]
```diff
@@ -563,26 +563,26 @@ def __init__(self, fig, rect,
...
 
         rect : [left, bottom, width, height]
-            The axes is build in the rectangle *rect*. *rect* is in
+            The Axes is build in the rectangle *rect*. *rect* is in
```
Comment (story645) [inline lib/matplotlib/axes/_base.py:574]:
```suggestion
            The Axes is built in the rectangle *rect*. *rect* is in
```

File: lib/matplotlib/axes/_base.py [hunk 3]
```diff
@@ -1002,7 +1002,7 @@ def _update_transScale(self):
 
     def get_position(self, original=False):
         """
-        Get a copy of the axes rectangle as a `.Bbox`.
+        Get a copy of the Axes rectangle as a `.Bbox`.
```
Comment (story645) [inline lib/matplotlib/axes/_base.py:1005]:
```suggestion
        Returns a copy of the Axes rectangle as a `.Bbox`.
```

File: lib/matplotlib/axes/_base.py [hunk 4]
```diff
@@ -2664,7 +2664,7 @@ def margins(self, *margins, x=None, y=None, tight=True):
         """
         Set or retrieve autoscaling margins.
 
-        The padding added to each limit of the axes is the *margin*
+        The padding added to each limit of the Axes is the *margin*
```
Comment (story645) [inline lib/matplotlib/axes/_base.py:2667]: Unclear what limit refers to here

**Review by story645 — APPROVED** [review]
Comments are mostly tangential to PR. Should Axis also be capitalized since it's also an object?

File: lib/matplotlib/axes/_base.py (hunk 3 above)
Comment (timhoffm) [reply lib/matplotlib/axes/_base.py]: Reworded. Copy is not the right word anyway.

**Comment by timhoffm** [comment]
> Should Axis also be capitalized since it's also an object?

+0.1. Technically yes, but for all other cases the class name and the logical concept are very close. So it's not that important, but still a lot of work. I'm not volunteering for that :).

File: lib/matplotlib/axes/_base.py (hunk 4 above)
Comment (timhoffm) [reply lib/matplotlib/axes/_base.py:2667]: I haven't a good description eiher.

Comment (QuLogic) [reply lib/matplotlib/axes/_base.py:2667]: each direction?

Comment (story645) [reply lib/matplotlib/axes/_base.py:2667]: cross reference against data limit? is it view_limit = data_limit + margin?

Comment (timhoffm) [reply lib/matplotlib/axes/_base.py:2667]: There's more to do in this context. I defer this to #20984 and merge this PR as is with only the Axes renaming.

**Pull request merged**